src/helpers/manage-merge-queue.ts

```diff
@@ -22,12 +22,18 @@ import {
 import { PullRequest } from '../types';
 import { context } from '@actions/github';
 import { map } from 'bluebird';
+import { notifyUser } from '../utils/notify-user';
 import { octokit } from '../octokit';
 import { removeLabelIfExists } from './remove-label';
 import { setCommitStatus } from './set-commit-status';
 import { updateMergeQueue } from '../utils/update-merge-queue';
 
-export const manageMergeQueue = async () => {
+interface ManageMergeQueue {
+  login?: string;
+  slack_webhook_url?: string;
+}
+
+export const manageMergeQueue = async ({ login, slack_webhook_url }: ManageMergeQueue) => {
```

Comment:
Nit pick, but maybe we could default the input so we don't need to update existing calls in the tests.
```suggestion
export const manageMergeQueue = async ({ login, slack_webhook_url }: ManageMergeQueue = {}) => {
```

Reply:
Updated, thank you!